components/gitlab/actions/create-new-merge-request-thread/create-new-merge-request-thread.mjs

```diff
@@ -0,0 +1,33 @@
+import gitlab from "../../gitlab.app.mjs";
+
+export default {
+  name: "Create New Merge Request Thread",
+  version: "0.0.1",
+  key: "create-new-merge-request-thread",
+  description: "Create a new thread on a Merge Request",
+  props: {
+    gitlab,
+    projectId: {
+      propDefinition: [
+        gitlab,
+        "projectId",
+      ],
+    },
+    mergeRequestIid: {
```

Comment:
extra `i` in the name

Reply:
@dylburger I know it's weird but that's the actual Gitlab variable name. Kind of funny, they have an `id` and an `iid` for the integer id.

Reply:
https://docs.gitlab.com/ee/api/discussions.html#create-new-merge-request-thread